.golangci.yml

```diff
@@ -242,6 +242,10 @@ issues:
     - ifshort
     text:  "variable 'isDeleteNodeAllowed' is only used in the if-statement.*"
     path: ^internal/controllers/machine/machine_controller\.go$
+  - linters:
+    - ifshort
+    text: "variable 'kcpMachinesWithErrors' is only used in the if-statement.*"
```

Comment:
how about the other kcpMachinesWith* variables in that file?

Reply:
For some reason this is the only one which is a false positive.

Additional info: It's not just a false positive, it's a flaky false positive. That is the reason why I couldn't use nolint. Because there is a nolint-linter which makes sure that every nolint matches to an actual finding which doesn't work with flaky false positives.